pkg/systemd/services.html

```diff
@@ -62,19 +62,15 @@
       <div class="filter-group">
         <button id="create-timer" data-toggle="#timer-dialog" data-target="#timer-dialog" class="btn btn-primary" translate>Create Timer</button>
         <input type="search" id="services-text-filter" class="form-control" aria-label="Filter" placeholder="Filter by name or description..." translate="placeholder"/>
-        <div class="dropdown" id="services-dropdown">
-          <button class="btn btn-default dropdown-toggle" type="button" id="service-type" data-toggle="dropdown">
-            <span id="current-service-type" class="pull-left" data-num=0 translatable="yes">All</span>
-            <span class="caret pull-right"></span>
-          </button>
-          <ul class="dropdown-menu" role="menu" aria-labelledby="service-type">
-            <li role="presentation"><a role="menuitem" data-num=0 translatable="yes">All</a></li>
-            <li role="presentation" class="divider"></li>
-            <li role="presentation"><a role="menuitem" data-num=1 translatable="yes">Enabled</a></li>
-            <li role="presentation"><a role="menuitem" data-num=2 translatable="yes">Disabled</a></li>
-            <li role="presentation"><a role="menuitem" data-num=3 translatable="yes">Static</a></li>
-          </ul>
-        </div>
+
+        <label id="services-dropdown-label" translatable="yes" class="hidden">Status</label>
+        <select class="ct-select" id="services-dropdown" aria-labelledby="services-dropdown-label">
+          <option value="0" translatable="yes" selected>All</option>
+          <option role="separator" className="divider" disabled="disabled">──────────</option>
```

Comment:
Lets probably drop this divider. It does not look that good, list it rather short, and also screen reader see it as element that is not accessible.

Reply:
Yeah. I wonder what other screen readers do too. Since it's the simplest option (ha! 😏) to remove it, this makes perfect sense.